src/main/menus/tray.js

```diff
@@ -29,6 +29,14 @@ function createTemplate(mainWindow, config) {
     }), {
       type: 'separator'
     }, {
+      label: 'Settings',
+      click: () => {
+        mainWindow.loadURL('file://' + __dirname + '/browser/settings.html');
+        mainWindow.show();
```

Comment:
Yes, it's for Windows and OS X. The problem reported in #249 was not reproduced in Linux (https://github.com/mattermost/desktop/pull/251#issuecomment-241278012).